bootstrap/scripts/2-download.sh

```diff
@@ -13,8 +13,8 @@ SCRIPTS="$(cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P)"
 
 mkdir -p "${BOOTSTRAP_CACHE}" #required to generate hermes files
 
-${BOOTSTRAP_REPOSITORY}/bootstrap/scripts/getPharoVM.sh 61
-wget --progress=dot:mega https://github.com/guillep/PharoBootstrap/releases/download/v1.5.0/bootstrapImage.zip
+${BOOTSTRAP_REPOSITORY}/bootstrap/scripts/getPharoVM.sh 70
+wget --progress=dot:mega https://github.com/carolahp/PharoBootstrap/releases/download/v1.7.0/bootstrapImage.zip
```

Comment:
Caro, it would be good if you make a PR to the original repository. We can move it under pharo-project/PharoBootstrap too.

Reply:
@guillep I did a [PR to your repo](https://github.com/guillep/PharoBootstrap/pull/8) 
By "moving it to pharo-project/PharoBootstrap", you mean to create a new repo?

Reply:
Thanks. I meant that we can transfer the repository from my user to pharo's user